lib/astarte_flow/blocks/device_events_producer/rabbitmq_client.ex

```diff
@@ -38,8 +38,15 @@ defmodule Astarte.Flow.Blocks.DeviceEventsProducer.RabbitMQClient do
     queue = Keyword.get(opts, :queue, "")
     connection = Keyword.get(opts, :connection, [])
     exchange = Keyword.get(opts, :exchange, "astarte_events")
+    prefetch_count = Keyword.get(opts, :prefetch_count, 100)
```

Comment:
We're defaulting to 300 everywhere, I'd keep it consistent.

Reply:
I'd use 100 here since we're bringing up `N` consumers, while in Astarte we (usually) bring up a single consumer